contracts/src/factory/factory.cairo

```diff
@@ -76,7 +81,12 @@ mod Factory {
         owner: ContractAddress,
         memecoin_class_hash: ClassHash,
         lock_manager_address: ContractAddress,
-        mut exchanges: Span<(SupportedExchanges, ContractAddress)>
+        mut exchanges: Span<(SupportedExchanges, ContractAddress)>,
+        core: ContractAddress,
```

Comment:
prefix the 3 by `ekubo_` so that we don't loose track of what it is

Reply:
done